EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -128,15 +183,173 @@ private void SetState(GuildDialogState state)
+                        {
+                            ShowIconBackGround = false,
+                            IconGraphic = _dialogIconService.IconSheet,
+                            IconGraphicSource = _dialogIconService.GetDialogIconSource(DialogIcon.GuildLeave),
+                            PrimaryText = _localizedStringFinder.GetString(EOResourceID.GUILD_LEAVE_GUILD),
+                            SubText = _localizedStringFinder.GetString(EOResourceID.GUILD_LEAVE_YOUR_CURRENT_GUILD),
+                            OffsetY = 48,
+                            OffsetX = -8,
+                        };
+                        leaveGuild.DrawArea = leaveGuild.DrawArea.WithSize(leaveGuild.DrawArea.Width + AdjustedDrawAreaOffset, leaveGuild.DrawArea.Height);
+                        leaveGuild.LeftClick += (_, _) => LeaveGuildDialog();
+                        leaveGuild.RightClick += (_, _) => LeaveGuildDialog();
+                        AddItemToList(leaveGuild, sortList: false);
+
+                        var registerGuild = new ListDialogItem(this, ListDialogItem.ListItemStyle.Large, 2)
+                        {
+                            ShowIconBackGround = false,
+                            IconGraphic = _dialogIconService.IconSheet,
+                            IconGraphicSource = _dialogIconService.GetDialogIconSource(DialogIcon.GuildRegister),
+                            PrimaryText = _localizedStringFinder.GetString(EOResourceID.GUILD_REGISTER_GUILD),
+                            SubText = _localizedStringFinder.GetString(EOResourceID.GUILD_CREATE_YOUR_OWN_GUILD),
+                            OffsetY = 48,
+                            OffsetX = -8,
+                        };
+                        registerGuild.DrawArea = registerGuild.DrawArea.WithSize(registerGuild.DrawArea.Width + AdjustedDrawAreaOffset, registerGuild.DrawArea.Height);
+                        //registerGuild.LeftClick += (_, _) => ShowGuildLookUpDialog();
+                        //registerGuild.RightClick += (_, _) => ShowGuildLookUpDialog();
+                        AddItemToList(registerGuild, sortList: false);
+                    }
+                    break;
+
+                case GuildDialogState.GuildBank:
+                {
+                    if (string.IsNullOrEmpty(_characterRepository.MainCharacter.GuildTag))
+                    {
+                        var dlg = _messageBoxFactory.CreateMessageBox(DialogResourceID.GUILD_NOT_IN_GUILD);
+                        dlg.ShowDialog();
+                        SetState(GuildDialogState.Initial);
+                    }
+                }
+                    break;
+            }
+        }
+
+        private void HandleDialogAction(string responseText)
+        {
+            switch (_currentDialogAction)
+            {
+                case DialogAction.GuildLookUp:
+                    Debug.WriteLine("Call Guild look up");
```

Comment:
If you're not familiar with the debugger, I'll write up a quick tutorial on helpful tips for using it. Scattering `Debug.WriteLine` everywhere through the code is not a sustainable solution

Reply:
Thanks! `Debug.WriteLine` is usually my go-to, and I'm somewhat familiar with the debugger, but a quick tutorial would still be very helpful. Appreciate it!